tests/e2e/page_tests/test_login.ts

```diff
@@ -1,38 +1,83 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 import { NightwatchBrowser } from "nightwatch";
 
+const url = process.env.VUE_DEV_SERVER_URL;
+
 const signInPage = {
-  "Verifying the atatSigninpage": (browser: NightwatchBrowser) => {
+  "Verifying the ATAT Signin page 1": (browser: NightwatchBrowser) => {
     const home = browser.page.atatHome();
 
-    home.navigate("http://localhost:8080").pause(10000);
+    home.navigate(url).pause(10000);
     home.expect.element("@classificationBanner").to.be.present.before(1000);
     home.expect
       .element("@classificationBanner_text")
       .text.to.equal("An official website of the United States government");
     home.expect
       .element("@classificationBanner_link_msg")
       .text.to.equal("Here’s how you know");
+    home.expect
+      .element("@atatImage")
+      .to.have.attribute("alt")
+      .which.startsWith("A")
+      .and.endsWith("TAT logo");
     home.expect.element("@atatText").text.to.contain("Access the ATAT Cloud");
+    home.assert.visible("@atatlogo");
     home.expect
       .element("@certSelectionText")
       .text.to.contain("Certificate Selection");
+    home.getText(
+      "@certSubText",
+      function (result: { status: any; value: any }) {
+        home.assert.equal(typeof result, "object");
+        home.assert.strictEqual(result.status, 0);
+        const certTextBody =
+          "When you are prompted to select a certificate, please select an\n" +
+          "Authentication (Identification) Certificate from the provided choices.";
+        home.assert.equal(result.value, certTextBody);
+      }
+    );
+    home.assert.visible("@lastLoginFooter");
+    home.getText(
+      "@lastLoginFooter",
+      function (result: { status: number; value: any }) {
+        home.assert.equal(typeof result, "object");
+        home.assert.strictEqual(result.status, 0);
+        console.log(result.value);
```

Comment:
Why are we logging the result?

Reply:
Due to the Dynamic changing of the value, if the test fails we will have a value in the log, so that we can debug the test.